tests/integ/test_df_to_pandas.py

```diff
@@ -117,8 +115,7 @@ def check_fetch_data_exception(query: str) -> None:
 
 
 @pytest.mark.skipif(
-    version.parse(snowflake.connector.__version__) < version.parse("2.7.5"),
-    reason="Python connector >= v2.7.5 ignores the index for the dataframe from fetch_pandas_all()",
+    IS_IN_STORED_PROC, reason="SNOW-507565: Need localaws for large result"
```

Comment:
We might be able to use `IS_IN_STORED_PROC_LOCALFS` here, again, orthogonal to your change and can be looked into separately